backend/.env.example

```diff
@@ -0,0 +1,19 @@
+# (Required) Database URL used for sqlx database connection
+DATABASE_URL=postgres://postgres:testpass@localhost:5432/testdb
+# This url corresponds to a docker postgres command as follows:
+# docker run --name testdb-container -e POSTGRES_PASSWORD=testpass -e POSTGRES_USER=postgres -e POSTGRES_DB=testdb -p 5432:5432 -d postgres
+
+# (Required) Allows sqlx macros to give compile-time errors
+SQLX_OFFLINE=true
```

Comment:
Don't think you need this for compile time errors, just for building without a DB connection